scripts/ssb_alert_manage

```diff
@@ -0,0 +1,109 @@
+##H Daemon for Alerting Tool (CERN SSB Status Board)
+##H Usage: ssb_alert_manage <Actions>
+##H
+##H Actions:
+##H   help        help manual
+##H   start       start the service
+##H   stop        stop the service
+##H   status      output service's status
+##H
+
+
+### ***
+#The following block of code was taken from
+#https://github.com/dmwm/deployment/blob/master/exitcodes/manage#L27
+
+ME=$(basename $(dirname $0))
+TOP=$(cd $(dirname $0)/../../.. && pwd)
+ROOT=$(cd $(dirname $0)/../.. && pwd)
+LOGDIR=$TOP/logs/$ME
+COLOR_OK="\\033[0;32m"
+COLOR_WARN="\\033[0;31m"
+COLOR_NORMAL="\\033[0;39m"
+
+### ***
+
+if [ ! -d $LOGDIR ]; then 
+  echo "Log Directory '$LOGDIR' does not exists. Creating..."
+  mkdir -p $LOGDIR
+fi
+
+if [ ! -w $LOGDIR ]; then 
+    echo "$USER has no write permission for '$LOGDIR' log directory. Exiting.."
+    exit 1
+fi
+
+## ssb_alert.sh arguments from Environment Variable
+QUERY=$query                                  ## monit influxDB query
+TOKEN=$token                                  ## token file
+URL=${url:-"https://cms-monitoring.cern.ch"}   ## alertmanager_url
+INTERVAL=${interval:-1}                       ## time interval at which fetching and injecting data repeats
+VERBOSE=${verbose:-0}                         ##verbosity level
+
+# Actions Logic
+start()
+{
+    echo "Starting Alerting daemon."
+    if [ ! -x "$(command -v rotatelogs)" ]; then
+      echo "rotatelogs not found. Starting without it..."
+      nohup ssb_alert.sh "$QUERY" $TOKEN $URL $INTERVAL $VERBOSE </dev/null 2>&1 | tee $LOGDIR/ssb_alerting-`date +%F`-`hostname -s`.log >/dev/null 2>&1 &
```

Comment:
why do you need `tee` to keep output on stdout? Please redirect directly to log file, that's it. If someone needs to see a progress than this person can use `tail -f <file.log>` for that. Basically you need to have the following:
```
nohup ssb_alert.sh "$QUERY" $TOKEN $URL $INTERVAL $VERBOSE 2>&1 1>& $LOGDIR/ssb_alerting-`date +%F`-`hostname -s`.log &
```

Reply:
My understanding was to redirect the stdout data to log file using `tee`. Thanks for the simpler approach. I have updated the line.